design-proposals/isolate-emulator-thread-with-smt-fix/isolate-emulator-thread-with-smt-fix.md

```diff
@@ -54,13 +54,63 @@ Fix the actual scheduling of the virt-launcher to a node that will reject it.
 This bug addresses all KubeVirt users running VMs with isolateEmulatorThread on nodes with hyperthreading enabled - specifically DPDK and Realtime users.
 
 ## User Stories
-A user should be able to run VM with dedicatedCPUPlancement and isolateEmulatorThread enabled on a node with hyperthreading enabled, regardless of the amount of guest CPUs requested.
+* As a cluster admin, I want to set the cluster-wide IsolatedEmulatorThread behavior for Kubevirt by setting a knob on HCO.
+* As a Kubevirt user, I should be able to run VM with dedicatedCPUPlancement and isolateEmulatorThread enabled on a node with hyperthreading enabled, regardless of the amount of guest CPUs requested.
 
 ## Repos
 - [kubevirt/kubevirt](https://github.com/kubevirt/kubevirt)
+- [kubevirt/hyperconverged-cluster-operator](https://github.com/kubevirt/hyperconverged-cluster-operator)
 
 # Bug fix design proposal
-The bug fix proposal introduces an alpha annotation `kubevirt.io/CPUManagerPolicyBetaOptions:full-pcpus-only`:
+The bug fix proposal introduces an alpha annotation to the kubevirt CR: `kubevirt.io/EmulatorThreadEvenParity`:
```

Comment:
Please consider `kubevirt.io/EmulatorThreadParityCompletion`.

Reply:
Or `evenVCpusForIsolatedEmulatedThread` as suggested by @dankenigsberg.

Reply:
how about `kubevirt.io/SetEvenVCPUsForEmulatedThread`?

Reply:
I personally prefer the first suggestion, but I think consulting a native speaker would help here.

Reply:
I asked @stu-gott, He said the either the suggested `kubevirt.io/EmulatorThreadEvenParity` or `kubevirt.io/EmulatorThreadCountEvenParity` might work. WDYT?

Reply:
chosen `kubevirt.io/EmulatorThreadCompleteToEvenParity`